scipy/stats/tests/test_stats.py

```diff
@@ -2190,8 +2190,8 @@ def test_scalar(self):
     def test_basic(self):
         data1 = [3, 5, 1, 10, 23, 3, 2, 6, 8, 6, 10, 6]
         vals = stats.mode(data1)
-        assert_equal(vals[0][0], 6)
-        assert_equal(vals[1][0], 3)
+        assert_equal(vals[0], 6)
+        assert_equal(vals[1], 3)
```

Comment:
The default behavior is no longer `keepdims`, so I'm removing a bunch of things like this.